policyengine_us/variables/gov/states/ma/tax/income/credits/ma_limited_income_tax_credit.py

```diff
@@ -20,12 +16,12 @@ def formula(tax_unit, period, parameters):
             "ma_income_tax_exemption_threshold", period
         )
         income_over_threshold = max_(0, agi - exemption_threshold)
-        income_ratio = agi / exemption_threshold
-        lic = parameters(
-            period
-        ).gov.states.ma.tax.income.credits.limited_income_credit
-        eligible = income_ratio <= lic.income_limit
-        tax_cap = lic.percent * income_over_threshold
+        income_ratio = np.ones_like(agi) * np.inf
```

Comment:
```suggestion
        # Compute AGI as a share of the exemption threshold.
        # Use a mask rather than where to avoid a divide-by-zero warning. Default to inf.
        income_ratio = np.ones_like(agi) * np.inf
```